notebooks/elasticsearch-spark-recommender.ipynb

```diff
@@ -891,6 +891,21 @@
     "        hits = results['hits']['hits']\n",
     "        return src, hits[1:num+1]\n",
     "    \n",
+    "def get_similar_users(the_id, q=\"*\", num=10, index=\"demo\", dt=\"users\"):\n",
```

Comment:
I believe we don't need a new method here. You can simply call `get_similar(id, q, num, index, dt="users")` for the same result.